### DISCUSSION TITLE
Images in imported e-mails

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#14694] [Wishlist]
Many of the e-mail I receive include small images in their signature. When the scanner imports these e-mails, it creates for each image a document in the Documents module. These images are annoying to me and of no use. Currently I am manually deleting the documents once in a while, but I would love to have this process automated.

I have no clue how this could be handled. I also have no idea how I would implement a solution if I had to do so. Maybe a first start would be to be able to associate a specific folder to documents found in e-mails imported?

Any ideas?

**Comment by rskrzypczak** [comment]
Each document created from the e-mail is assigned to the **Mails** folder
![image](https://user-images.githubusercontent.com/10221999/118918353-eff13d80-b932-11eb-9533-37183e3b57ec.png)

**Reply by diderich** [reply]
Thanks. Is that value "Mails" configured somewhere? I did not see it in `config/Modules/OSSMailScanner.php`.

**Reply by rskrzypczak** [reply]
This is a tree field, it is not set in the config file
https://gitdeveloper.yetiforce.com/index.php?module=TreesManager&parent=Settings&view=Edit&record=1

**Reply by diderich** [reply]
Does that mean that the second item in that tree is the one to which the mail scanner documents are assigned, or how does the system know which field to assign to mail related documents?

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/blob/1cbc7da44666bdc8f17145f45ed8dfdeda7040ea/modules/OSSMailScanner/scanneractions/CreatedEmail.php#L126

**Reply by diderich** [reply]
Thanks @mariuszkrzaczkowski for pointing this out! Maybe this could be moved to the configuration `config/Modules/OSSMailScanner.php` in an upcoming release, e.g. by adding a constant `public static $documentFolderId = 'T2'` referenced in `modules/OSSMailScanner/scanneractions/CreatedEmail.php`?

**Reply by s2b-git** [reply]
It would be of an advantage if scanner omit in-line embedded images, but consider only attached.

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/commit/8b39361c9b649972e9f41100340c31cbdbcff157
